dracut/99setup-root/initrd-setup-root-after-ignition

```diff
@@ -134,6 +134,7 @@ for NAME in $(grep -h -o '^[^#]*' /sysroot/etc/flatcar/enabled-sysext.conf /sysr
     echo "Did not find ${ACTIVE_EXT}" >&2
     systemctl start --quiet systemd-networkd systemd-resolved
     download_and_verify "flatcar-${NAME}.raw"
+    mkdir -p "/sysroot/etc/flatcar/sysext/"
```

Comment:
For this folder and the others such as `/sysroot/etc/flatcar/oem-sysext/` that we don't have in the underlay we could add a comment here that creating it makes it an opaque overlay folder. This means that the underlay won't be used if we want to ship something by default in the future (which we shouldn't because we can also read files from /usr).
```suggestion
    # Note: This folder is created opaque since it does not exist in the underlay
    # For cases where we ship it later, it's best to add a "attr" call in flatcar-postinst
    # because there are many ways how the folder might have been created
    mkdir -p "/sysroot/etc/flatcar/sysext/"
```

Reply:
added the comment